Header header
fcsc/ObjectMask[] object_mask_arr
